pkg/controller/pxc/users.go

```diff
@@ -455,6 +468,23 @@ func (r *ReconcilePerconaXtraDBCluster) manageOperatorAdminUser(cr *api.PerconaX
 	return nil
 }
 
+func (r *ReconcilePerconaXtraDBCluster) manageReplicationUser(cr *api.PerconaXtraDBCluster, sysUsersSecretObj *corev1.Secret) error {
+	addr := cr.Name + "-pxc." + cr.Namespace
+	if cr.CompareVersionWith("1.6.0") >= 0 {
+		addr = cr.Name + "-pxc-unready." + cr.Namespace + ":33062"
+	}
+	um, err := users.NewManager(addr, "root", string(sysUsersSecretObj.Data["root"]))
```

Comment:
why we user root here? I think it should be operator user
```suggestion
	pxcUser := "root"
	pxcPass := string(internalSysSecretObj.Data["root"])
	if _, ok := internalSysSecretObj.Data["operator"]; ok {
		pxcUser = "operator"
		pxcPass = string(internalSysSecretObj.Data["operator"])
	}
	um, err := users.NewManager(addr, pxcUser, pxcPass)
```

Reply:
33249c2